core/tx_pool.go

```diff
@@ -1162,3 +1159,68 @@ func (as *accountSet) containsTx(tx *types.Transaction) bool {
 func (as *accountSet) add(addr common.Address) {
 	as.accounts[addr] = struct{}{}
 }
+
+// txLookup is used to track all transactions to allow lookups without contention
```

Comment:
Please end method/type docs with punctuation marks, they should be complete sentences.

Reply:
Also, I'd add an emphasis as to why this type exists and how it should be used.

```
// txLookup is used to track all transactions to allow lookups without contention.
//
// Note, although this type is properly protected against concurrent access, it
// is **not** a type that should ever be mutated or even exposed outside of the
// transaction pool, since its internal state is tightly coupled with the pools
// internal mechanisms. The sole purpose of the type is to permit out-of-bound
// peeking into the pool without interfering with internal reorgs.
```